idea/testData/intentions/convertLambdaToReference/suspendFunctionParameter3.kt

```diff
@@ -0,0 +1,7 @@
+fun foo(a: suspend () -> Unit) {}
```

Comment:
By default this conversion will lead to red code, because currently it is disabled

Please enable it with this

```suggestion
// COMPILER_ARGUMENTS: -XXLanguage:+SuspendConversion
fun foo(a: suspend () -> Unit) {}
```